apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -288,6 +290,292 @@ static void itc_wifi_manager_connect_disconnect_ap_p(void)
+	strncpy(config.ssid, TEST_SSID, config.ssid_length + 1);
+	strncpy(config.passphrase, TEST_PASSWORD, config.passphrase_length + 1);
+	config.ap_auth_type = (wifi_manager_ap_auth_type_e)TEST_AUTH_TYPE;
+	config.ap_crypto_type = (wifi_manager_ap_crypto_type_e)TEST_CRYPTO_TYPE;
+	printf("AP config: %s(%d), %s(%d), %d %d\n", config.ssid, config.ssid_length, config.passphrase, \
+			config.passphrase_length, config.ap_auth_type, config.ap_crypto_type);
+
+	ret =  wifi_manager_save_config(&config);
+	TC_ASSERT_EQ_CLEANUP("wifi_manager_save_config", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
+
+	ret =  wifi_manager_get_config(&getconfig);
+	TC_ASSERT_EQ_CLEANUP("wifi_manager_get_config", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
+
+	ret = strncmp(config.ssid, getconfig.ssid, strlen(TEST_SSID));
+	TC_ASSERT_EQ_CLEANUP("string cmp between SSID ", ret, 0, wifi_manager_deinit());
+
+	ret = strncmp(config.passphrase, getconfig.passphrase, strlen(TEST_PASSWORD));
+	TC_ASSERT_EQ_CLEANUP("string cmp between PASSWORD ", ret, 0, wifi_manager_deinit());
+
+	ret = wifi_manager_deinit();
+	TC_ASSERT_EQ("wifi_manager_deinit", ret, WIFI_MANAGER_SUCCESS);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @testcase         itc_wifi_manager_get_config_n
+ * @brief            check initialize, connect, save ap config, get ap config, de-initalization operations of wifi manager
+ * @scenario         check initialize, connect, save ap config, get ap config, de-initalization operations of wifi manager
+ * @apicovered       wifi_manager_init, wifi_manager_save_config, wifi_manager_get_config,, wifi_manager_deinit
+ * @precondition     none
+ * @postcondition    none
+ */
+static void itc_wifi_manager_get_config_n(void)
+{
+	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
+	ret = wifi_manager_init(&wifi_callbacks);
+	TC_ASSERT_EQ("wifi_manager_init", ret, WIFI_MANAGER_SUCCESS);
+
+	wifi_manager_ap_config_s config;
+	wifi_manager_ap_config_s gconfig;
+
+	config.ssid_length = strlen(TEST_SSID);
+	config.passphrase_length = strlen(TEST_PASSWORD);
+	strncpy(config.ssid, TEST_SSID, config.ssid_length + 1);
+	strncpy(config.passphrase, TEST_PASSWORD, config.passphrase_length + 1);
+	config.ap_auth_type = (wifi_manager_ap_auth_type_e)TEST_AUTH_TYPE;
+	config.ap_crypto_type = (wifi_manager_ap_crypto_type_e)TEST_CRYPTO_TYPE;
+	printf("AP config: %s(%d), %s(%d), %d %d\n", config.ssid, config.ssid_length, config.passphrase, \
+			config.passphrase_length, config.ap_auth_type, config.ap_crypto_type);
+
+	ret =  wifi_manager_get_config(&gconfig);
+	TC_ASSERT_NEQ_CLEANUP("wifi_manager_get_config", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
+
+	ret = wifi_manager_deinit();
+	TC_ASSERT_EQ("wifi_manager_deinit", ret, WIFI_MANAGER_SUCCESS);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @testcase         itc_wifi_manager_get_remove_config_p
+ * @brief            check initialize, connect,, save ap config, get ap config, remove ap config, de-initalization operations of wifi manager
```

Comment:
there are no connect API on the function. please check the comment on ITC testcases